src/routes/generative-ai/+page.svelte

```diff
@@ -1,86 +1,258 @@
 <script lang="ts">
 	import LandingHero from '../components/landing-hero.svelte';
-	const title = 'Generative AI + ONNX Runtime';
+	import { fade } from 'svelte/transition';
+	import coffee from './coffee.png';
+	import genny from './genny.png';
+	import vision_ui from './vision_UI.png';
+	import mobile from './mobile.png';
+	import desktop from './desktop.png';
+	import cloud from './browser.png';
+	import aibrain from './aibrain.webp'
+	const title = 'Generative AI';
 	const description =
-		'Integrate the power of generative AI in your apps and services with ONNX Runtime. Broad platform support and deep optimizations empower usage of state-of-the-art models for image synthesis, text generation, and more.';
+		'Integrate the power of generative AI in your apps and services with ONNX Runtime. Broad platform support and deep optimizations empower usage of state-of-the-art models for text generation, audio synthesis, and more.';
```

Comment:
Do we want to keep the meta description and first para technology generic?

Reply:
will specify more directly in the meta